ElectronicObserver/Window/Wpf/ShipTrainingPlanner/ShipTrainingPlanViewModel.cs

```diff
@@ -22,6 +23,8 @@ public partial class ShipTrainingPlanViewModel : ObservableObject
 
 	public event Action? OnSave;
 
+	public bool ShipRemodelLevelReached => TargetRemodel is IShipDataMaster remodel && Ship.Level >= remodel.RemodelAfterLevel;
```

Comment:
todo : add check that remodel != ship.mastership
and allow null to be selected in the combobox for targeted remodel